user-service/models/users.js

```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose')
```

Comment:
Entity is always singular. Change from users to user

Reply:
Yes will change that